src/functionalTest/resources/features/F-004/S-045.td.json

```diff
@@ -0,0 +1,39 @@
+{
+  "_guid_": "S-045",
+  "_extends_": "F-004_Test_Base_data",
+  "title": "must get an error response for a non-existing secutiry classification",
+  "specs": [
+    "contains a non-existing secutiry classification"
+  ],
+  "request": {
+    "headers": {
+      "caseTypeId": "BEFTA_CASETYPE_2_2",
+      "jurisdictionId": "BEFTA_JURISDICTION_2"
+    },
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "classification",
+          "value": "NonExisting"
+        },
+        {
+          "key": "roles",
+          "value": "caseworker"
+        },
+        {
+          "key": "files",
```

Comment:
"files" --> "file"

Reply:
same as @aashishaggarwal123  above comment i.e. This upload document endpoint support multiple documents. thats why its key is files by default.